### DISCUSSION TITLE
Plans to add Ordnance Survey maps?

### DISCUSSION THREAD

**Discussion by steveg271** [peterLaurence/TrekMe#71] [Ideas]
Are there any plans to add OS 50K and 25K to the app?

Thanks

**Comment by p-lr** [comment]
To my knowledge, the API used by TrekMe is now the only one accessible to new users:
https://www.ordnancesurvey.co.uk/business-government/tools-support/ondemand-map-tile

"OS OnDemand Web Map Tile Service is no longer available for new users (being withdrawn in November 2021). We have now launched a new suite of APIs, accessible from our new data access platform, OS Data Hub."

I don't know if they plan to add OS 50K and 100K to their API. If they do, I'll certainly add them.
